sea-query-binder/src/sqlx_postgres.rs

```diff
@@ -117,6 +129,188 @@ impl<'q> sqlx::IntoArguments<'q, sqlx::postgres::Postgres> for SqlxValues {
                 Value::MacAddress(mac) => {
                     args.add(mac.as_deref());
                 }
+                #[cfg(feature = "postgres-array")]
+                Value::Array(ty, v) => match ty {
+                    ArrayType::Bool => {
+                        let value: Option<Vec<bool>> =
+                            Value::Array(ty, v).except("Invalid type for array value");
+                        args.add(value)
+                    }
```

Comment:
I think ordinary user won't see this message (unless someone intentionally construct a vector of boxed value with different value variants in it) but the error message itself can be more specific.

```rs
Value::Array(ty, v).except("This Value::Array should consist of Value::Bool");
```

Reply:
Done